src/components/About/About.module.css

```diff
@@ -3,4 +3,23 @@
   font-family: "Roboto", "Helvetica", "Arial", sans-serif;
   color: #ccc;
   margin: 10px 0;
-}
\ No newline at end of file
+}
+.error {
+  font-family: "Roboto", "Helvetica", "Arial", sans-serif;
+  padding: 10px 10px 10px 44px;
+  font-size: 14px;
+  line-height: 20px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected single space after "," in a single-line function (function-comma-space-after)](https://app.codacy.com/gh/WebHeroSchool/agatta17_todo/pullRequest?prid=5682719)